packages/pkp-base/src/lib/pkp-base.ts

```diff
@@ -234,16 +239,14 @@ export class PKPBase<T = PKPBaseDefaultParams> {
       );
     }
 
-    // Print deprecation warning for controllerSessionSigs
-    if (this.controllerSessionSigs) {
-      logError(
-        'controllerSessionSigs is deprecated, please use authContext instead'
-      );
-    }
-
-    // Check auth context if provided
+    // Check if authContext is provided
     if (this.authContext) {
-      // It must have a valid client and getSessionSigsProps
+      // Try to assign litNodeClient to authContext.client if it's not already set and litNodeClient is available
```

Comment:
What's the difference in authContext in case of an eoaSessionSig and pkpSessionSigs?

Reply:
in `authContext` you need to create your own authNeededCallaback, but if you use our abstracted function `getPkpSessionSigs`, then you can just pass in the `sessionSigs` to the `PkpEthersWallet`

Reply:
Can the user pass authMethods for their PKP here instead of a sessionSig? Or they have to get their PKP sessionSig first and then pass that?

Reply:
moving this conversation to https://github.com/LIT-Protocol/js-sdk/pull/459#discussion_r1596980924